monkey/monkey_island/cc/ui/src/components/Main.js

```diff
@@ -96,6 +97,11 @@ class AppComponent extends AuthComponent {
     };
   }
 
+  static isReportUrl(){
+    let url = window.location.href;
+    return ( url.endsWith('/report/security') || url.endsWith('/report/attack') || url.endsWith('/report/zeroTrust'))
+  }
```

Comment:
Not happy about it, but this is the only solution I could think of without redux